proto/v1/setting_service.proto

```diff
@@ -293,3 +294,19 @@ message DataClassificationSetting {
   }
   repeated DataClassificationConfig configs = 1;
 }
+
+message DataCategorySetting {
+  message DataCategoryItem {
+    // id is the uuid for category item.
+    string id = 1;
+    // the title of the category item, it should not be empty.
+    string title = 2;
+    // the description of the category item, it can be empty.
+    string description = 3;
+    // We do not support custom algorithm by now, we only support the default algorithm, so we do not
+    // add the algorithm field right now.
+  }
+
+  // items is the map of id - DataCategoryItem.
```

Comment:
What's the `id` here, is it classification id?

Reply:
It is the id of the `DataCategoryItem`, the reason of using map instead of the array is to prevent duplicate ids from being set when the api is called artificially. BTW, I add the comments to make it more clearly.